lib/ui/painting/picture.cc

```diff
@@ -74,9 +81,12 @@ void Picture::RasterizeToImageSync(sk_sp<DisplayList> display_list,
         sk_sp<SkImage> sk_image;
         std::string error;
         std::tie(sk_image, error) = snapshot_delegate->MakeGpuImage(
-            display_list, dl_image->dimensions());
+            display_list, dl_image->dimensions(), dl_image.get());
```

Comment:
What I really want here is a weak pointer, but SkWeakRefCnt is private and converting this to a std::shared_ptr is involved.